packages/common/src/chaindata/chaindata.ts

```diff
@@ -329,9 +325,9 @@ export class ChainData {
    * @param account The account to check.
    * @returns The identity root string.
    */
-  getIdentity = async (account: string): Promise<string | null> => {
-    return await getIdentity(this, account);
-  };
+  // getIdentity = async (account: string): Promise<string | null> => {
```

Comment:
why there is some commented code here and in other sections of the PR ? can we remove it ?

Reply:
Yes - commenting these out was a quick thing in getting building fixed to get staging back up. They can be removed as they aren't used anywhere. Should be removed in the next commit.